demo/articles/arabic.html

```diff
@@ -52,7 +52,7 @@
             </p>
         </div>
         <!-- Scripts -->
-        <script src="../wikipedia-preview.development.js"></script>
+        <script src="/dist/wikipedia-preview.umd.cjs"></script>
```

Comment:
we are not able to use this in `npm run dev`, this is only available when we run `npm run build`

the solution can be script target="module" to import it to use; then when we deploy to demo site, we need to replace it to normal script (either umd or iife format)

Reply:
How about setting `"dev": "vite build && vite",` in the package.json scripts?

Reply:
that's right, this will work like what we have in webpack version, https://github.com/wikimedia/wikipedia-preview/pull/177/commits/adc9d1c6b3f51cc8d3f55bc9967c1b9f21573c44